clase3/colegio.ts

```diff
@@ -0,0 +1,22 @@
+import { Alumno } from "./alumno";
+import { Profesor } from "./profesor";
+
+export class Colegio{
+    nombre:string;
+    alumno1 = new Alumno("andres", 7);
+    alumno2 = new Alumno("paula",6);
+    alumnos : Alumno[] = [this.alumno1,this.alumno2];
+    profesores:Profesor[]=[new Profesor("sandra"), new Profesor("alejandro"), new Profesor("jose")]
+    constructor(){
+        this.nombre = this.nombre;
```

Comment:
`this.nombre = this.nombre` no va a tener ningun valor. Dale un valor por defecto o hace que lo reciba por parámetro en el constructor